package.json

```diff
@@ -33,27 +33,32 @@
     "@openstfoundation/mosaic-tbd": "https://github.com/OpenSTFoundation/mosaic-tbd.git"
   },
   "devDependencies": {
-    "chai": "4.1.2",
-    "mocha": "5.0.0",
     "@babel/core": "7.1.0",
     "@babel/polyfill": "7.0.0",
     "@babel/preset-env": "7.1.0",
     "@babel/runtime": "7.0.0",
+    "abi-decoder": "1.2.0",
     "babel-loader": "8.0.2",
+    "chai": "4.1.2",
     "commander": "2.8.1",
     "edit-json-file": "1.0.8",
+    "eslint": "^5.13.0",
```

Comment:
Remove carrot sign `^`. We always specify exact version.
Apply above for other packages.

Reply:
I opened an issue on the new "developer-guidelines" to discuss this.  On mosaic.js we have not followed this for `eslint`. https://github.com/OpenST/developer-guidelines/issues/1